srcpkgs/googlemaps/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'googlemaps'
+pkgname=googlemaps
```

Comment:
I think package name should include `qtplugins`, otherwise it's slightly confusing.

Mageia calls it `qtlocation-plugin-googlemaps`, though that seems a bit long.

Reply:
Done